test/metrics/basic_test.rb

```diff
@@ -21,9 +21,8 @@
       # our test series reported (as above)?
 
       found_episode_plays = false
-      tbody = find('metrics-downloads-table div.table-wrapper table tbody')
+      tbody = find('metrics-downloads-table div.table-wrapper table.scroll-x tbody')
       tbody.all('tr').each do |tr|
-        next unless tr.text.match("Test Episode #{random_str}")
```

Comment:
The episode title is now in a separate `table.sticky` and the data is in table.scroll-x.